library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -34,6 +34,7 @@
     private static final String[] PROJECT_BATCH_ID = {DownloadContract.Batches._ID};
     private static final String WHERE_DELETED_VALUE_IS = DownloadContract.Batches.COLUMN_DELETED + " = ?";
     private static final String[] MARKED_FOR_DELETION = {"1"};
+    private static final String SORT_BY_ID_DESCENDING = DownloadContract.Batches._ID + " DESC";
```

Comment:
but from a UI perspective the second item goes to queued, which are shown below running ones e.g.:

Before check:
- Download 1 (will get denied) : queued
- Download 2 (won't get denied) : queued
- Download 3 (won't get denied) : queued

After check:
- Download 2 : running
- Download 1 : queued
- Download 3 : queued